blurb/tests/pass/no-break-on-hyphens.rst.res

```diff
@@ -1,5 +1,5 @@
-.. bpo: 21121
```

Comment:
What happens if someone adds `.. bpo: xxxx` instead of `.. issue:`?

Reply:
I've made another commit that still supports `bpo-` news files. This is because `blurb merge` tool still need to be able to read both types of news files, `bpo-` and `gh-issues`.